### PR TITLE
docs: Add a doc for NG5014

### PR DESCRIPTION
**Pull request by JeanMeche**
Short explanation this help fix the `Root node not found` error.

I got the idea when I saw #50564.

### CODE REVIEW

File: aio/content/errors/NG5104.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+@name Root node not found.
```
Comment (AndrewKushnir) [inline aio/content/errors/NG5104.md:1]:
```suggestion
@name Root element was not found during bootstrap.
```

File: aio/content/errors/NG5104.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+@category runtime
+@shortDescription Root node was not found.
```
Comment (AndrewKushnir) [inline aio/content/errors/NG5104.md:3]:
```suggestion
@shortDescription Root element was not found during bootstrap.
```

File: aio/content/errors/NG5104.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```
+Replace the tag with the correct one: 
+
+```html
+<html>
+    <my-app></my-app> <!-- OK -->
+</html>
+```
+
+2. A component that shouldn't is being bootstraped
```
Comment (AndrewKushnir) [inline aio/content/errors/NG5104.md:37]: It's a bit unclear what do we want to communicate here. Which components would be considered unsupported in this case?
```suggestion
2. An unsupported component being bootstrapped
```

Comment (JeanMeche) [reply aio/content/errors/NG5104.md:37]: I got the idea from #50564 where OP erronously added a webcomponent in the bootstrap property. I not sure that conveys the message.

File: aio/content/errors/NG5104.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+<html>
+    <my-app></my-app> <!-- OK -->
+</html>
+```
+
+2. An unsupported component being bootstrapped
+
+```typescript
+@NgModule({
+  bootstrap: [AppComponent, DetailComponent], // DetailComponent is not a root component
```
Comment (AndrewKushnir) [inline aio/content/errors/NG5104.md:41]: I'm still a bit confused under which circumstances this can be a problem...

Comment (AndrewKushnir) [reply aio/content/errors/NG5104.md:41]: Yeah, it feels like an extension of the 1st point that talks about CSS selector mismatch. Let's remove this for now and expand the docs later as needed (if there are other use-cases when this error can happen).

**Review by AndrewKushnir — APPROVED** [review]
👍

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: public-api

**Comment by AndrewKushnir** [comment]
**Caretaker note**: low risk change, no extra reviews are required. We can also skip presubmit for this change. This PR is ready for merge (once CI is "green").

**Review by dylhunn — APPROVED** [review]
reviewed-for: public-api

**Comment by pkozlowski-opensource** [comment]
@JeanMeche it looks like the `test_aio` [failures](https://app.circleci.com/pipelines/github/angular/angular/61874/workflows/25db6ff2-9a3b-463a-ae6d-4edd8e3433ee/jobs/1345353) are real as those don't go away after restart.

Could you PTAL?

**Comment by JeanMeche** [comment]
The aio tests are flaky, I'll try a rebase.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 0522f1b35355d4772f70fa941c7c566d84236316.

**Pull request closed without merging**